app/views/bright_events/_filters.html.erb

```diff
@@ -24,7 +24,7 @@
 
   <script>
     function updateMinDate() {
-      var dateFromValue = document.getElementById('date_from').value;
+      let dateFromValue = document.getElementById('date_from').value;
       document.getElementById('date_to').min = dateFromValue;
     }
   </script>
```

Comment:
This isn't the way to do J's in rails!!!!

Reply:
Sorry, we will correct it as soon as possible.